.github/workflows/promote.yaml

```diff
@@ -63,7 +63,6 @@ jobs:
           clusterctl generate cluster --from https://github.com/kubernetes-sigs/cluster-api-provider-openstack/releases/download/v0.12.4/cluster-template-without-lb.yaml --worker-machine-count 1 test | kubectl apply -f -
         env:
           OS_CLOUD: "devstack"
-          KUBERNETES_VERSION: "1.33.4"
```

Comment:
I think we shouldn't just remove it but instead use it from the image that's being tested.

Reply:
Assuming the tag syntax remains the same, semantic version extraction changed to use GitHub `ref_name.`